.github/actions/test_ios_rntester/action.yml

```diff
@@ -0,0 +1,158 @@
+name: test_ios_rntester
+description: Test iOS RNTester
+inputs:
+  jsengine:
+    description: Which JavaScript engine to use. Must be one of "Hermes", "JSC".
+    default: Hermes
+  use-frameworks:
+      description: The dependency building and linking strategy to use. Must be one of "StaticLibraries", "DynamicFrameworks"
+      default: StaticLibraries
+  architecture:
+    description: The React Native architecture to Test. RNTester has always Fabric enabled, but we want to run integration test with the old arch setup
+    default: NewArch
+  ruby-version:
+    description: The version of ruby that must be used
+    default: 2.6.10
+  run-unit-tests:
+    description: whether unit tests should run or not.
+    default: false
+  hermes-tarball-artifacts-dir:
+    description: The directory where the hermes tarball artifacts are stored
+    default: /tmp/hermes/hermes-runtime-darwin
+  flavor:
+    description: The flavor of the build. Must be one of "Debug", "Release".
+    default: Debug
+  hermes-version:
+    description: The version of hermes
+    required: true
+  react-native-version:
+    description: The version of react-native
+    required: true
```

Comment:
from where these came from? It could be that I'm not super familiar with GH actions, but does this mean that who invoke the `test_ios_RNTester` job has to pass these inputs?
How can they know which values need to be passed?

Reply:
These values are [outputs of the `prepare_hermes_workspace` job](https://github.com/facebook/react-native/blob/77efd191a8293bea88f3753f0404c10d37e39de5/.github/workflows/test-all.yml#L40). Specifically within that job, the [`Setup hermes version` step](https://github.com/facebook/react-native/blob/77efd191a8293bea88f3753f0404c10d37e39de5/.github/workflows/test-all.yml#L48) gets the hermes version, and the [`Get react-native version` step](https://github.com/facebook/react-native/blob/77efd191a8293bea88f3753f0404c10d37e39de5/.github/workflows/test-all.yml#L63C15-L63C39) gets the react-native version.  

These outputs are used to pass in these values [here](https://github.com/robandpdx-org/react-native/blob/4a90c3f16735e4e3eb2db6d7d31e4bcd4fd0de33/.github/workflows/test-all.yml#L370). These values are needed for cache keys.